index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 global.D2LAccessTokenExpiresAt = 0;
+global.D2LOAuth2Disabled = false;
```

Comment:
I'm not 100% sure I remember why we did it but some guesses:

- Communicating across multiple copies of this library
   - Dubious: shouldn't do that and also the savings are low.
- Makes testing possible
   - Dubious, but what would be the most JS'y way to do this? add some properties onto module.exports?